packages/sync-actions/src/tax-categories-actions.js

```diff
@@ -30,11 +30,18 @@ export function actionsMapRates(diff, oldObj, newObj) {
       action: 'removeTaxRate',
       taxRateId: objectToRemove.id,
     }),
-    [CHANGE_ACTIONS]: (oldObject, updatedObject) => ({
-      action: 'replaceTaxRate',
-      taxRateId: oldObject.id,
-      taxRate: updatedObject,
-    }),
+    [CHANGE_ACTIONS]: (oldObject, updatedObject) =>
+      oldObject.id === updatedObject.id
+        ? {
```

Comment:
This could be simpler I think. It's quite noisy

```js
[CHANGE_ACTIONS]: (oldObject, updatedObject) => ({
  action: 'replaceTaxRate',
  taxRateId: oldObject.id === updatedObject.id ? oldObject.id : updatedObject.id,
  taxRate: updatedObject,
})
```

Reply:
Good catch --> e494503